code/modules/mod/mod_actions.dm

```diff
@@ -0,0 +1,37 @@
+/datum/action/item_action/mod
+	background_icon_state = "bg_tech_blue"
+	icon_icon = 'icons/mob/actions/actions_mod.dmi'
+	var/obj/item/mod/control/mod
+
+/datum/action/item_action/mod/New(Target)
+	..()
+	mod = Target
+
+/datum/action/item_action/mod/Grant(mob/M)
+	if(owner)
+		return
+	..()
+
+/datum/action/item_action/mod/deploy
+	name = "Deploy MODsuit"
+	desc = "Deploy/Conceal a part of the MODsuit."
+	button_icon_state = "deploy"
+
+/datum/action/item_action/mod/deploy/Trigger()
+	mod.choose_deploy(usr)
```

Comment:
are you trying to go for an "suiting up sequence" vibe here? if not, could you add a "deploy all parts" button or something?

Reply:
yea